src/app.js

```diff
@@ -0,0 +1,13 @@
+import Router from './router';
+
+class App {
+  constructor(container) {
```

Comment:
사소하지만 DOM 요소인지를 명확하게 드러내기 위해서 요런 이름을 쓰기도 합니다~
```suggestion
  constructor($container) {
```